npm/postinstall.js

```diff
@@ -0,0 +1,52 @@
+const stat = require("fs").stat
+const spawn = require("child_process").spawn
+const join = require("path").join
+const pkg = require("../package.json")
+
+console.log(pkg.name, "post-install", process.cwd())
+
+stat("lib", function(error, stats1) {
+  if (!error && stats1.isDirectory()) {
+    return true
+  }
+
+  console.warn(
+    "\n" +
+    "Builded sources not found. It looks like you might be attempting " +
+    `to install ${ pkg.name } from git. \n` +
+    "Sources need to be transpiled before use. This may take a moment." +
+    "\n"
+  )
+
+  const spawnOpts = {
+    stdio: "inherit",
+    cwd: join(__dirname, "../"),
+  }
+
+  const fail  = (err) => {
```

Comment:
The user won't notice that. The report a bug immedialy when they saw it. (which is related to node version).